### PR TITLE
Code style enforcement

### PR DESCRIPTION
**Pull request by wusatosi**
Trying out airbnb's code style template

**_This will be ugly_**

Discussed changes:
- [x] Change to double quote
- [x] Change single line character limit

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -15,7 +17,14 @@
       <div class="tab-content card-body">
         <div class="tab-pane fade show active" id="projects">
           <div class="card-text">
-            <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#add-project-modal"><i class="fas fa-plus"></i> Create Project</button>
```
Comment (wusatosi) [inline src/views/Home.vue:18]: This line now has to be warped around, is this what we want?
Does this look better?

File: src/views/Home.vue [hunk 2]
```diff
@@ -30,21 +39,44 @@
...
                   </div>
                   <div class="modal-footer">
-                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
-                    <button type="submit" form="createProject" class="btn btn-primary" name="projectCreationSubmit">Create</button>
+                    <button
+                      type="button"
+                      class="btn btn-secondary"
+                      data-dismiss="modal"
+                    >
+                      Cancel
```
Comment (wusatosi) [inline src/views/Home.vue]: Here, it looks kinda "off"

**Comment by wusatosi** [comment]
eslint should work now, and it can reformat everything upon `npm run lint` if it is able to.
I'll open this pr to review

**Comment by wusatosi** [comment]
However, there's some code style change here that I don't find particularly pleasing.
(see above)


~~And its mainly surrounding the fact that the HTML portion of vuejs single-file component may exceed 100 char limit.~~
~~Although there are places where it improves readability,~~
~~but there are also opposite examples where it got warped around and looks kind awful 😢~~

~~We have following possible path to resolve this:~~
~~1. Continue with this code style~~
~~2. Disable eslint check for HTML parts (I don't know if this is possible)~~
~~3. Disable maximum character per line restriction (or make it raise a warning instead of error)~~

After reading [Dashboard.vue](https://github.com/Visualizd/client/blob/abffa332d9cf0221021196ef552887d54d786b35/src/components/Dashboard.vue) (a section where tens of eslint errors were raised),
I think this code style got a point,
the html do seem to be to chaotic, I'd suggest componentize that part,
you'll prob have less indentation so its harder to reach line length limit.

I made some comments on abffa332d9cf0221021196ef552887d54d786b35 , please check it out and let's see which part should we lean to.

**Comment by wusatosi** [comment]
Please do look around with this pull request and make suggestions on the code style

**Comment by wusatosi** [comment]
todo: change to double quote

**Review by williamjallen — COMMENTED** [review]
Is this ready to be merged?

**Pull request merged**